app/controllers/bookings_controller.rb

```diff
@@ -23,27 +26,29 @@ def create
   end
 
   def show
+    authorize @booking
   end
 
   def edit
+    authorize @booking
   end
 
   def update
     @booking.update(booking_params)
```

Comment:
I think we also need to authorize the update as well

Reply:
thank you. add the authorization to the update method